distribution/changelog.txt

```diff
@@ -14,6 +14,7 @@
 - Change: [#7248] Small mini-maps are now centred in the map window.
 - Change: [#20240] Heavy snow and blizzards now make guests buy and use umbrellas.
 - Change: [#21214] Wacky Worlds and Time Twister’s scenario names now match their park names.
+- Change: [#22057] Reorder Time Twister's scenarios and adjust their difficulty classification.
```

Comment:
Should not have been a typewriter quote. (The entry above uses a correct apostrophe.)